packages/wallet-sdk/src/store/store.ts

```diff
@@ -93,6 +94,16 @@ const createSubAccountConfigSlice: StateCreator<StoreState, [], [], SubAccountCo
   };
 };
 
+type SpendLimitsSlice = {
+  spendLimits: Record<string, SpendLimit[]>;
```

Comment:
should be a number

Reply:
seems like all other numerical field in store are number type, let me update

Reply:
i wonder if we should establish a convention
1. only to use hex string for external interface, ie EIP
2. if a numerical field is used only internally, use `number`

cc: @stephancill wanna get y'all thoughts

Reply:
updated to nummmmm